### PR TITLE
Move XMP to core spec

### LINKED ISSUE #70: Bring the XMP text from the Extensions doc into the core spec
https://w3c.github.io/PNG-spec/extensions/Overview.html#K.XMP

### PR DESCRIPTION
**Pull request by svgeesus**
Fixes #70

### CODE REVIEW

File: extensions/Overview.html [hunk 1]
```diff
@@ -568,38 +551,38 @@ <h3 class="heading settled" data-level="4.2" id="C.pCAL"><span class="secno">4.2
...
      <tr>
-      <th>Null separator:
-      <td>1 byte
+      <th>Null separator: 
+      <td>1 byte 
      <tr>
-      <th>Parameter 1 (p1):
-      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>)
+      <th>Parameter 1 (p1): 
+      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>)
```
Comment (ProgramMax) [inline extensions/Overview.html:585]: Was adding the trailing spaces intentional?

File: extensions/Overview.html [hunk 2]
```diff
@@ -775,7 +758,7 @@ <h3 class="heading settled" data-level="4.2" id="C.pCAL"><span class="secno">4.2
     <a class="self-link" href="#example-e3c29ac5"></a> For example,
         floating-point data ranging from
         -10<sup>31</sup> to 10<sup>31</sup> (the usual range of 32-bit floating-point numbers)
-        can be represented with:
+        can be represented with:
```
Comment (ProgramMax) [inline extensions/Overview.html:761]: (Same here)

File: extensions/Overview.html [hunk 3]
```diff
@@ -2012,16 +1985,16 @@ <h3 class="heading settled" data-level="12.2" id="Credits.contrib"><span class="
...
+    png @ adeluc.com 
+    <li>Brendan Bolles 
+    <li>Todd French, <code>tfrench @ sandia.gov</code> 
+    <li>Phil Harvey 
+    <li>Nancy M. Randers-Pehrson 
+    <li>Alaric B. Snell, <code>alaric @ alaric-snell.com</code> 
+    <li>Michael Stokes 
+    <li>Jason Summers 
+    <li>Cosmin Truta, <code>cosmin @ cs.toronto.edu</code> 
+    <li>Pavel Zlatovratskii
```
Comment (ProgramMax) [inline extensions/Overview.html:1997]: (Same here)

File: index.html [hunk 4]
```diff
@@ -4026,23 +4026,23 @@ <h2><span class="chunk">cICP</span>
...
 
-  <p>Note: In Coding-Independent Code Points 
-    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>, the 
+  <p>Note: In Coding-Independent Code Points
+    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>, the
     third video coding point defines matrix coefficients which are used to describe
-    conversions from other colour representations to RGB. PNG image samples are 
-    explicitly RGB, so the matrix coefficients value must be set to "0" which is 
-    labeled in <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a> 
-    as "RGB Identity". With PNG, decoders will always assume a matrix coefficient
```
Comment (ProgramMax) [inline index.html:4045]: Same here. Was removing the trailing space intentional?

(We should probably run this doc through a formatter at some point.)

**Review by ProgramMax — APPROVED** [review]
LGTM.
Some nits about trailing whitespaces that probably weren't intentional. But it looks like we should run these files through a formatter anyway, so I'm not worried about them.

File: extensions/Overview.html (hunk 1 above)
Comment (svgeesus) [reply extensions/Overview.html:585]: I have no idea why the trailing whitespace appeared. It is only on the bikeshed-generated output. My editor is set to strip trailing whitespace (which sometimes gives diff problems)

**Pull request merged**